### PR TITLE
Use Apache httpd rather than httpd for better SEO

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: src/content/1.7/scale/webservers/_index.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ It is often said that nginx is faster than httpd, which is not entirely true. Ng
 If you want to know more about configuring [nginx for PrestaShop](../webservers/nginx/).
 
 
-#### httpd
+####  httpd/apache2
```
Comment (PierreRambaud) [inline src/content/1.7/scale/webservers/_index.md:57]:
```suggestion
####  Apache httpd
```
Use the realname instead: https://httpd.apache.org/

**Comment by matks** [comment]
@PierreRambaud Thanks ! applied

File: src/content/1.7/scale/webservers/_index.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ It is often said that nginx is faster than httpd, which is not entirely true. Ng
 If you want to know more about configuring [nginx for PrestaShop](../webservers/nginx/).
 
 
-#### httpd
+####  Apache httpd
```
Comment (PierreRambaud) [inline src/content/1.7/scale/webservers/_index.md:57]:
```suggestion
#### Apache httpd
```

File: src/content/1.7/scale/webservers/httpd.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Optimize your httpd configuration
+title: Optimize your httpd/apache2 configuration
```
Comment (PierreRambaud) [inline src/content/1.7/scale/webservers/httpd.md:2]:
```suggestion
title: Optimize your Apache httpd configuration
```

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**